http_client.rst

```diff
@@ -627,6 +627,11 @@ according to the ``multipart/form-data`` content-type. The
         'body' => $formData->bodyToIterable(),
     ]);
 
+By default, :class:`Symfony\\Component\\Mime\\Part\\Multipart\\FormDataPart`
+will transfer file uploads in binary encoding. But you can enforce Base64::
+
+    $fileField = DataPart::fromPath('/path/to/uploaded/file', null, null, 'base64');
```

Comment:
```suggestion
    $fileField = DataPart::fromPath(path: '/path/to/uploaded/file', encoding: 'base64');
```

Reply:
Let's linebreak the parameters for easier reading